### DISCUSSION TITLE
I have my school email but github asks me to add anotehr one

### DISCUSSION THREAD

**Discussion by bisrattsegaye12** [community/community#138691] [GitHub Education] [answered]
### Hi everyone,

Hello,

I am a student at Hawassa University and our student email addresses are in the format: studentname@hustudents.edu.et. However when I try to apply for GitHub's student benefits it requests an email in the format: studentname@hu.edu.et.

When I contacted the responsible department at my university they confirmed that the correct and current email format for students is @hustudents.edu.et.

Could you advise how I can apply using the email address provided by my university?

Error message while applying:
![image](https://github.com/user-attachments/assets/d229c30b-c6ea-45c4-93dd-aa8b82f35bd3)

Email format my university gave me:
![image](https://github.com/user-attachments/assets/aa6edc20-dd21-4161-8295-70ed976df515)

P.S. The year in the second image Ethiopian calendar.

Thank you!

**Comment by davevad93** [comment, marked as answer]
Hi @bisrattsegaye12 ,

You should contact support and inform them that the current domain of your academic institution that appears on their list isn't the correct one. Raise a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain the issue with your academic email address.
